NoSQL/README.md

```diff
@@ -5,50 +5,68 @@ Data is all around us and knowing how to use and manipulate databases is an incr
 ## The Goal 🥅
 **Who?** Built with the audience of college and high school students that have minimal to advanced coding experience in mind. Most do not have experience with NoSQL databases and deploying web applications.
 
-**What?** Concise and interactive Github Learning Lab that introduces and develops skills related to NoSQL databases with [MongoDB](https://azure.microsoft.com/en-us/services/functions/)
+**What?** Concise and interactive Github Learning Lab that introduces and develops skills related to NoSQL databases with [MongoDB](https://www.mongodb.com/)
+
+**How?** Students gain experience through building a complete web app with a frontend (HTML, CSS, JS, Atlas Realms) and backend (MongoDB, Azure Services, NodeJS). The application utilizes machine learning, an external API, and NoSQL databases to recommend products to users based off of a picture. Authentication is used to incorporate the database, which stores a history of recommendations with a corresponding user.
```

Comment:
...databases to recommend products to users based on the contents in an uploaded image (objects and text within the picture).